### PR TITLE
Added QTest support to Protobuf files in DQMFileSaverPB

### PR DESCRIPTION
**Pull request by andrius-k**
#### PR description:

Adds QTest information to protocol buffer files saved with DQMFileSaverPB.

#### PR validation:

PR was validated locally.

### CODE REVIEW

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
please test

File: DQMServices/FileIO/plugins/DQMFileSaverPB.cc [hunk 1]
```diff
@@ -236,8 +236,26 @@ void DQMFileSaverPB::savePB(DQMStore* store, std::string const& filename, int ru
     histo.set_size(buffer.Length());
     histo.set_streamed_histo((void const*)buffer.Buffer(), buffer.Length());
 
-    // Save quality reports if this is not in reference section.
-    // XXX not supported by protobuf files.
+    // Save quality reports
+    for (QReport* qr : me->getQReports()) {
```
Comment (silviodonato) [inline DQMServices/FileIO/plugins/DQMFileSaverPB.cc:240]: `for (const auto& qr : me->getQReports()) { `
if you are just reading `qr`

**Comment by silviodonato** [comment]
what's the status/plan of this PR? 
@andrius-k @jfernan2 @kmaeshima @fioriNTU @schneiml

**Comment by andrius-k** [comment]
Hi Silvio @silviodonato,
This PR is related to the new DQM GUI that is in the works. It can be merged on its own without waiting for the work on the new DQM GUI to be completed.

**Comment by silviodonato** [comment]
merge

File: DQMServices/FileIO/plugins/DQMFileSaverPB.cc (hunk 1 above)
Comment (andrius-k) [reply DQMServices/FileIO/plugins/DQMFileSaverPB.cc:240]: As promised, implemented this here: https://github.com/cms-sw/cmssw/pull/30592/files#diff-7b79ea209bca0428f38b6b7786f2220dR266

**Comment by andrius-k** [comment]
+1
Signing for the sake of completeness.

**Pull request merged**